model/train.py

```diff
@@ -40,6 +40,7 @@
 import jax.numpy as jnp
 import ml_collections
 import numpy as np
+import pickle as pkl
```

Comment:
Nit: This should appear with the standard library imports.  (https://google.github.io/styleguide/pyguide.html#313-imports-formatting)